support/src/main/java/se/inera/intyg/common/support/facade/model/user/LoginMethod.java

```diff
@@ -19,5 +19,5 @@
 package se.inera.intyg.common.support.facade.model.user;
 
 public enum LoginMethod {
-    BANK_ID, SITHS, FAKE
+    BANK_ID, SITHS, FAKE, BANK_ID_MOBILE
```

Comment:
What is the difference between BANK_ID and BANK_ID_MOBILE for the purpose of signing?

Reply:
The only difference is really only some copy. This is taken directly from the old client https://github.com/sklintyg/frontend/pull/484/files#diff-b86c34ac30f1f81a6658a0ebf86ab6b77ec35455c92490732eb83ecb0fd49addR13-R17

Reply:
OK! Then I understand. We want to give different messages to the user depending if they are using MOBILE or DESKTOP BANKID.